src/main/resources/web_en.properties

```diff
@@ -44,10 +44,14 @@ fileUpload.URL = URL
 fileUpload.specifyURL = Enter URL
 footer-menu.html.help = Help System
 help = Help
+selectableTable.all.deselection = Deselect all
+selectableTable.all.selection = Select all
+selectableTable.row.deselection = Deselect
+selectableTable.row.selection = Select
 table.html.empty = No contents found
 template.html.additionalActions = Additional Actions
 template.html.confirmHeader = Are you sure?
-template.html.confirmText = Press 'Yes' to continue or Cancel to go back.
+template.html.confirmText = Press 'Yes' to continue or 'Cancel' to go back.
```

Comment:
This line was only adapted in English, not in the other languages. What about the German version?

Reply:
`Drücken Sie 'Ja', um fortzufahren oder 'Abbrechen', um zurückzugehen.`

Reply:
The english version has been enhanced by the quotes. The German original already came with quotes (although not the correct ones... but we need to fix the quotation in sirius anyway as this is not the only key)

Reply:
Look at _all_ the other languages as well, plz. There, the new quotes seem to be missing.

Reply:
Yes, they are missing there indeed.